packages/react-core/src/deprecated/components/Tile/examples/Tile.md

```diff
@@ -1,15 +1,80 @@
 ---
 id: Tile
 section: components
+cssPrefix: pf-v5-c-tile
+propComponents: ['Tile']
+deprecated: true
 ---
 
+import PlusIcon from '@patternfly/react-icons/dist/esm/icons/plus-icon';
+import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
+import './Tile.css';
+
+Note: Tile has been deprecated, please use the [Card](/components/card) component.
```

Comment:
Could we point to the specific tile instance here `/card#cards-as-tiles`

Reply:
@andrew-ronaldson AFAIK we haven't added any styles to the card to mimic the hint component - does the "hint" card need to look like anything in particular, or should it just be a selectable card? In the [cards as tiles example](https://patternfly-react-pr-10821.surge.sh/components/card/#cards-as-tiles), I _think_ the selectable styles should be there, I'm guessing it's a bug with the example. But we don't support an image/icon in a card specifically like we do for the hint. You can put an image/icon in there, but you'll need to style it yourself, and space/align it relative to the text unless we want to expand the card styles to support that.